Cargo.toml

```diff
@@ -25,24 +32,33 @@ rand = { version = "0.8", features = ["small_rng"] }
 rand_distr = "0.4"
 order-stat = "0.1"
 puruspe = "0.2"
-matrixmultiply = { version = "0.3", features = ["threading"] }
+matrixmultiply = { version = "0.3", features = ["threading", "cgemm"] }
```

Comment:
Regarding the matrixmultiply dependency in Cargo.toml:

The current specification includes the "cgemm" feature unconditionally:
matrixmultiply = { version = "0.3", features = ["threading", "cgemm"] }

However, the "cgemm" feature is already correctly tied to Peroxide's "complex" feature in the features section:
[features]
complex = [ ..., "matrixmultiply/cgemm"]

Suggestion:
Remove the "cgemm" feature from the direct dependency specification:
matrixmultiply = { version = "0.3", features = ["threading"] }

Reply:
Done in PR: https://github.com/Axect/Peroxide/pull/71/commits/f2778957a20f24e9ef709482a4a7e0013c1b7788